.github/pull-request-template.md

```diff
@@ -1,4 +1,11 @@
-Fixes #(issue-number)
+Fixes #issue-number
+
+<!-- Most PRs should be opened against the master branch.
+
+If the change should also be in the most recent release, add the
+cherrypick-0.X label, for example, cherrypick-0.5. If you can, open a PR for the
```

Comment:
```suggestion
corresponding "cherrypick-0.X" labels to the PR and indicate which branch(es) your change applies to, for example, "cherrypick-0.5". Best practice is to open a PR for the
```

Reply:
I added the "corresponding" wording and the quotes around the specific label names. I would rather keep the "If you can" wording, because I don't want to discourage people who don't feel comfortable with git from contributing.

Reply:
I feel the same too which is why i used "best practice" but I also was to prevent us from having to do the maintenance work (especially if it ends up being P3 and we should probably not spend time on P3 given the backlog of P1). Just thinking long term here (today we definitely want everything shiny and perfect, but I dont think we can keep that up for long).

Reply:
Okay, I switched the wording to "best practice."